utils/unittest/blocks.go

```diff
@@ -0,0 +1,32 @@
+package unittest
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// RequireContainBlock is a test helper that fails if specified blocks list does not contain the
+// specified blocks.
+func RequireContainBlock(t *testing.T, block *flow.Block, blocks []*flow.Block) {
```

Comment:
Why we need these test helpers?

When we store a block fixture into the database (i.e., `storage.Blocks`) it encodes all empty structs of it to _nil_. Examples of such empty member fields can be a block with empty receipts. This causes the following assertion to fail on the tests _for identical blocks_: 
In the following code snippet, the `actualBlock` is the original block we store to `storage.blocks`, and `retrievedBlock` is the block we retrieve from` storage.Blocks`

```
require.Equal(t, actualBlock, retrievedBlock) // fails because some members (e.g., payload) are empty but encoded as nil on storage
```

As a solution, we came up asserting blocks based on their block ids: 

```
require.Equal(t, actualBlock.ID(), retrievedBlock.ID()) // this passes
```

But then it prevents us from comparing two lists of blocks (i.e., original vs retrieved from storage).
```
require.MatchElements(t, blocks[:3], received) // this fails even for same blocks. 
```
Hence, we developed these test helper logics ☝️ 👇  to compare blocks lists based on ids.